.env.example

```diff
@@ -1,15 +1,60 @@
 APP_NAME=lmt
-ENVIRONMENT=dev
+APP_ENV=local
+APP_KEY=base64:URqZeYUM/VChKM2nRWIMhz1kZ3Y4nyMKlxz4yB3WWng=
```

Comment:
Should it be here?

Reply:
I don't see a problem. This is just an random app key so you don't have to generate it by yourself.

Reply:
Fixed